packages/coachmark/coach-indicator.md

```diff
@@ -23,7 +23,21 @@ When looking to leverage the `CoachIndicator` base class as a type and/or for ex
 import { CoachIndicator } from '@spectrum-web-components/coachmark';
 ```
 
-## Static color variants
+### Examples
```

Comment:
Can we change the heading to Anatomy here to be consistent with other docs? And maybe just note that it consists of a single animated visual indicator.